pkg/virt-api/webhooks/validating-webhook/admitters/vmi-preset-admitter.go

```diff
@@ -30,6 +30,7 @@ import (
 
 	v1 "kubevirt.io/api/core/v1"
 
+	storageAdmitters "kubevirt.io/kubevirt/pkg/storage/admitters"
```

Comment:
Please do not use the camel case naming for the imports.
We usually use lowercase

Reply:
I wrote this here, but it is valid for all the other occurrences

Reply:
sure ill fix it